README.md

```diff
@@ -1 +1,60 @@
-#spring-boot-remote-git-autoconfigure
+spring-boot-remote-git-autoconfigure
+=======================================
+It can download the Spring Boot configurations from remote git repo and load the configuration to Spring boot APP
+
+## Prerequisites
+You should install the git client successfully and the configure the ssh auto verify with remote git repo successfully. It mean that you can use the OS git command pull the remote configuration repo item successfully.
+
+## Add the dependency jar 
```

Comment:
Take Singleton service and client's readme as reference and from users' perspective, is it better to add a parent title 'How to build and use it'? 

And in this part add content about how to build it.

Reply:
I tend to think that 'how to build' belong to how to contribute part. It only introduces 'how to use it'

Reply:
That's ok. My main point is 'how to use it' contains below 4 steps:
## Add the dependency jar 
## create the remote git config repository for Spring Boot apps
## Add the configuration item to application configuration(application.properties, application.yml, application.yaml)
## Start the spring boot app
right? Make each step as first level heading looks ungraceful, so had better add 'how to use it' as first level heading which parallel with 'Prerequisites', and change them as secondary heading under 'how to use it', please refer to https://github.com/vmware/singleton/tree/g11n-java-client readme.
how do you think?

Besides, will you add contribution file in this PR?